README.md

```diff
@@ -120,6 +120,91 @@ Dane będa przesyłane jako strumień i przetwarzane zgodnie z typem wiadomości
 - Dodaj regulator
 - Dodaj urządzenie
 
+## API
+```text
+// undefined oznacza, że może nie być tego w requeście
+{
+  //GET /devices - zwrotka
+  "devices": [
+    {
+      "id": "string",
+      "regulatorId": "string",
+      "name": "string",
+      "status": "ENUM: ACTIVE | INACTIVE | INVALID | CONNECTING",
+      "type": "ENUM: TEMPERATURE | LIGHT",
+      "data": "string | boolean",
+      "targetData": "string | boolean"
+    }
+    // ...
+  ],
+
+  //POST /devices - request klienta
+  "body": {
+    "regulatorId": "string",
+    "name": "string",
+    "publicKey": "BASE64 encoded string"
+  },
+
+  //PATCH /devices/:deviceId request klienta
+  "body": {
+    "regulatorId": "string | undefined",
+    "name": "string | undefined",
+    "publicKey": "BASE64 encoded string | undefined"
+  },
+  // ...,
+
+  //DELETE /devices/:deviceId
+  "devices request": "puste body",
+
+  // POST /devices/setTargetData
+  "target data request": {
+    "id": "string",
+    "targetData": "string | boolean"
+  },
+
+  //GET /regulators - tak powinna wyglądać zwrotka
+  "regulators": [
+    {
+      "id": "string",
+      "name": "string",
+      "status": "ENUM: ACTIVE | INACTIVE | INVALID | CONNECTING",
+      "type": "ENUM: TEMPERATURE | LIGHT"
+    }
+    // ...
+  ],
+
+  //POST /regulators request klienta
+  "body": {
+    "regulatorId": "string",
+    "name": "string",
+    "publicKey": "BASE64 encoded string"
+  }
+
+  //PATCH /regulators/:regulatorId request klienta
+  "body": {
+    "type": "ENUM: TEMPERATURE | LIGHT",
+    "name": "string | undefined",
+    "publicKey": "BASE64 encoded string | undefined"
+  },
+
+  // DELETE /regulators/:deviceId
+  "regulators request": "puste body"
```

Comment:
rozumiem, że tutaj nie ma body?

Reply:
to samo do przykładu nieco wyżej

Reply:
nie wiedziałem jak to zapisać, ale tak tu nie ma nic w response body

Reply:
i w request body też nic nie ma bo wystarczy to bie tylko to że jest to regulator i `:deviceId` jest w urlu